czech/admin/config_languages.lang.php

```diff
@@ -18,7 +18,7 @@
 $l['edit_with'] = "Změnit s {1}";
 $l['edit_language_variables'] = "Upravit jazykové mutace";
 $l['edit_language_variables_desc'] = "Zde můžete upravit jazykové řetezce dle svých představ.";
-$l['edit_properties_desc'] = "Zde můžeš změni vlastnosti jazykového balíčku.";
+$l['edit_properties_desc'] = "Zde můžete změnit vlastnosti jazykového balíčku.";
 $l['installed_language_packs'] = "Nainstalované jazykové mutace";
 $l['front_end'] = "Frontend";
```

Comment:
Co toto, má to vube nejaky cesky vyraz?

Reply:
Nepokoušel bych se překládat úplně všechno doslova. Za mě je Frontend v pořádku.
Taky nepřekládáme backdoor jako zadní vrátka. :-)